website/docs/d/database.html.markdown

```diff
@@ -89,4 +89,4 @@ In addition to all argument references list, you can access the following attrib
 
 
 **Note**
-The provider only exports the admin user ID and associated connection string. It does not export any user IDs that are configured for the instance in addition. 
```

Comment:
Did you mean to make a change here?

Reply:
I did not but cant see what the change actually is!

Reply:
Got it, was new line change